akka-stream-tests/src/test/scala/akka/stream/impl/TimeoutsSpec.scala

```diff
@@ -130,6 +130,105 @@ class TimeoutsSpec extends AkkaSpec {
 
   }
 
+  "BackpressureTimeout" must {
+
+    "pass through elements unmodified" in assertAllStagesStopped {
+      Await.result(Source(1 to 100).backpressureTimeout(100.millis).grouped(200).runWith(Sink.head), 3.seconds) should === (1 to 100)
+    }
+
+    "succeed if backpressure notified" in assertAllStagesStopped {
+      val subscriber = TestSubscriber.probe[Int]()
+
+      Source(1 to 4)
+        .backpressureTimeout(100.millis)
```

Comment:
Increase it a bit everywhere, perhaps `500ms` as on jenkins such tight timeouts end up being fleaky